packages/appstore/src/components/add-more-wallets/__test__/add-more-wallets.spec.tsx

```diff
@@ -0,0 +1,150 @@
+    it('should render the component without errors', () => {
+        const mock = mockStore({
+            client: {
+                loginid: 'CRW909900',
+                accounts: {
+                    CRW909900: {
+                        token: '12345',
+                    },
+                },
+                is_crypto: (currency: string) => currency === 'BTC',
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <APIProvider>
+                <StoreProvider store={mock}>{children}</StoreProvider>
+            </APIProvider>
+        );
+
+        render(<AddMoreWallets />, { wrapper });
+
+        expect(screen.getByTestId('dt-add-wallets')).toBeInTheDocument();
+    });
+
+    it('should render the title correctly', () => {
+        const mock = mockStore({
+            client: {
+                loginid: 'CRW909900',
+                accounts: {
+                    CRW909900: {
+                        token: '12345',
+                    },
+                },
+                is_crypto: (currency: string) => currency === 'BTC',
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <APIProvider>
+                <StoreProvider store={mock}>{children}</StoreProvider>
+            </APIProvider>
+        );
+
+        render(<AddMoreWallets />, { wrapper });
+        expect(screen.getByText('Add more Wallets')).toBeInTheDocument();
+    });
+
+    it('should render the carousel', () => {
+        const mock = mockStore({
+            client: {
+                loginid: 'CRW909900',
+                accounts: {
+                    CRW909900: {
+                        token: '12345',
+                    },
+                },
+                is_crypto: (currency: string) => currency === 'BTC',
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <APIProvider>
+                <StoreProvider store={mock}>{children}</StoreProvider>
+            </APIProvider>
+        );
+
+        render(<AddMoreWallets />, { wrapper });
+        expect(screen.getByTestId('dt-carousel-container')).toBeInTheDocument();
+    });
+
+    it('should render the wallet add card', () => {
+        const mock = mockStore({
+            client: {
+                loginid: 'CRW909900',
+                accounts: {
+                    CRW909900: {
+                        token: '12345',
+                    },
+                },
+                is_crypto: (currency: string) => currency === 'BTC',
+            },
+        });
+
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <APIProvider>
+                <StoreProvider store={mock}>{children}</StoreProvider>
+            </APIProvider>
+        );
+
+        render(<AddMoreWallets />, { wrapper });
+        const wallet_cards = screen.queryAllByTestId('dt-add-wallet-card');
```

Comment:
if you check this [document](https://testing-library.com/docs/queries/about/#priority) you will see that getByTestId has the lowest priority in the all the selectors. we can still achieve the same thing by using `getByText` here

Reply:
@ameerul-deriv since we are mocking this, there is not much difference between selectors, but overally you are correct.